submissions/YuliiaHordiichuk/popup/css/style.css

```diff
@@ -0,0 +1,215 @@
+body { 
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px;
+    color: rgb(117,117,117); 
+    font-weight: normal;
+    font-style: normal; 
+}
+
+.header {
+    width: 100%; 
+    background-color: rgba(128, 128, 128, 0.1);
+}
+
+.header__nav{
+    position: relative; 
+    width: 80%;
+    margin-right: auto;
+    margin-left: auto; 
+    padding: 20px 0; 
+}
+
+/* Main nav */
+
+.nav__list {
+    display: flex; 
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.nav__item {
+    margin-right: 25px; 
+}
+
+.nav__item:last-child {
+    margin-right: 0;
+}
+
+.nav__link {
+    display: flex; 
+    align-items: center;
+    justify-content: center;
+    padding: 5px; 
+
+    color: inherit; 
+    text-decoration: none;
+    border: 1px solid transparent;
+
+    transition: .2s background;
+}
+
+.nav__icon {
+    width: 25px; 
+    object-fit: contain;
+}
+
+.nav__link:hover,
+.nav__checkbox:hover,  
+.popup__label:hover, 
+.popup__footer:hover, 
+.popup__link:hover {
+    background-color:rgba(3, 5, 14, 0.1); 
+    border-radius: 5px;
+}
+
+.popup__label:hover, 
+.popup__footer:hover {
+    border-radius: 0;
+}
+
+.nav__link:focus,
+.nav__checkbox:focus,  
+.popup__link:focus, 
+.popup__footer:focus, 
+.popup__checkbox:focus,
+.popup__checkbox:focus + .popup__label {
+    outline: none; 
+    border: 1px solid rgba(128, 128, 128, 0.5);
+    border-radius: 5px;
+}
+
+.nav__checkbox {
+    width: 37px; 
+    height: 37px; 
+
```

Comment:
the good practice - do not leave an empty line between rules. Could you please fix it for all cases?